src/components/atoms/UserAvatar/UserAvatar.module.scss

```diff
@@ -0,0 +1,19 @@
+@use "scss/attendee/layout";
+
+.avatarContainer img {
+  width: 100%;
+  height: 100%;
+}
+
+@each $sizeName, $sizePixels in layout.$avatarSizes {
+  .avatarContainer__#{$sizeName} {
+    width: $sizePixels;
+    height: $sizePixels;
```

Comment:
If there is a need of a list just for the sake of looping over it, it can be made in this file as a private constant without bleeding it through the layout module to all other components.

These can be left out or remixed, classnames function will deal with undefined:

```scss
$-sizes: ( 
  "small": layout.$sz-avatar--sm,
  "medium": layoot.$sz-avatar--md,
  "large": layout.$sz-avatar--lg,
  "xlarge": layoot.$sz-avatar--xl,
);

@each $name, $size in $-sizes {
  .#{name} {
    width: $size;
    height: $size;
```